.env.regtest

```
@@ -18,10 +18,7 @@ ARK_CONTAINER=ark
 # ── Boltz override ─────────────────────────────────────────────────────────
-# regtest/.env.defaults pins BOLTZ_IMAGE=boltz/boltz:latest. That tag was
-# re-pushed on 2026-05-12 at 15:51 UTC with a 3.13.0-7ae3002e dev build that
-# breaks BTC→ARK chain-swap settlement (CanDoBtcToArkChainSwap times out;
-# arkd batch rounds cascade-fail with "not enough intent confirmations
-# received"). Pin to the v3.13.0 stable tag — same major version, without
-# the post-release dev commits.
-BOLTZ_IMAGE=boltz/boltz:v3.13.0
+# We're overriding the image with a release born from https://github.com/BoltzExchange/boltz-backend/pull/1467
+# so we can test covclaimd
+BOLTZ_IMAGE=boltz/boltz:ark-claim 
+COVCLAIMD_IMAGE=ghcr.io/arkade-os/covclaimd:v0.0.1-rc.1 
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)